kinematics_simulator_node:
  simulator:
    init_x: 0.0
    init_y: 0.0
    init_yaw: 0.0
    time_resolution: 0.01
    max_velocity: 2.0
    max_angle_velocity: 2.4
    time_delay: 0.0
    noise_std: 0.0
